### ISSUE TITLE
Minor issues with cancel/returns

### ISSUE THREAD

**Issue by Xymmer** [SAP/spartacus#6722]
Cancel/returns feature is in Spartacus now but officially not usable until Commerce 2005 release.

Some issues seen testing recently that might have already been reported and/or might require backend fix:
- language code displayed instead of status text, after cancelling
![image](https://user-images.githubusercontent.com/4998145/75355735-f2263280-587c-11ea-8b8b-67e13e68711b.png)
- "Pending" status for "consignment" that appears for items cancelled, before consignments actually created (so it's just a bucket). (1) If all items in "pending" are cancelled, status should be "Cancelled". (2) Shouldn't display promo's if all items in a consignment (or in "Pending" non-consignment bucket) are cancelled, or if consignment itself is cancelled.
![image](https://user-images.githubusercontent.com/4998145/75355954-429d9000-587d-11ea-8dbe-825892702b4b.png)

**Comment by bgambocjaviniar** [comment]
QA: 

- place an order
- go to the order history of that order

verify the unconsigned entry label is `Pending` when it's processing
verify the unconsigned entry label is `Cancelled` when it's cancelled

*Note
- to cancel an order, use a 2005 server, such as dev7 as it has order/returns/cancellations enabled.

**Comment by KateChuen** [comment]
Tested on dev3 (frontend) dev7 (backend).
✅verify the unconsigned entry label is Pending when it's processing
<img width="1180" alt="Screen Shot 2020-05-06 at 9 04 27 AM" src="https://user-images.githubusercontent.com/45409515/81181992-3b7cb780-8f7b-11ea-8dd7-2e982f511a87.png">

✅verify the unconsigned entry label is Cancelled when it's cancelled
<img width="1217" alt="Screen Shot 2020-05-06 at 9 05 10 AM" src="https://user-images.githubusercontent.com/45409515/81182001-3fa8d500-8f7b-11ea-9370-e230ef2a9fc1.png">

What have we decided about this point that Bill mentioned?
(2) Shouldn't display promo's if all items in a consignment (or in "Pending" non-consignment bucket) are cancelled, or if consignment itself is cancelled.
